ompi/runtime/ompi_rte.c

```diff
@@ -743,6 +743,14 @@ int ompi_rte_init(int *pargc, char ***pargv)
         val = NULL;
     }
 
+    /* get our initial errhandler */
+    val = NULL;
+    OPAL_MODEX_RECV_VALUE_OPTIONAL(rc, "MPI_INITIAL_ERRHANDLER",
```

Comment:
I don't understand where this gets set - it appears to be some MPI-specific value, but that means PRRTE will never set it, and there is no way to successfully receive it here. Can you add some comments explaining who and where this gets set, and what the values are supposed to be?

Reply:
Thanks, I removed it, it was a leftover from a prior version.

The env variable that replaces it is set from PRUN (ompi_schizo)

Reply:
If ompi/schizo knows what the value should be, then there is no issue with getting it from PMIx and prun - it just wasn't clear to me how it was being generated and how prrte would know the right value.